src/editors/EditorFooter.jsx

```diff
@@ -65,7 +65,7 @@ export default function EditorFooter() {
               : (
                 <FormattedMessage
                   id="authoring.editorfooter.savebutton.label"
-                  defaultMessage="Add To Course"
+                  defaultMessage="Save"
```

Comment:
Blocks could be saved to libraries instead of just the course so this button is now changed to "Save".